Cargo.toml

```diff
@@ -57,6 +57,7 @@ tectonic_xdv = { path = "xdv", version = "0.1.9-dev" }
 termcolor = "^1.0"
 toml = "^0.5"
 zip = { version = "^0.5", default-features = false, features = ["deflate"] }
+atty = "0.2.13"
```

Comment:
I try to keep these lists alphabetized, so this new entry should go near the top. Also, can we just specify the version as "0.2"?